app/views/orders/new.html.erb

```diff
@@ -0,0 +1,161 @@
+<div id="all">
+  <div id="content">
+    <div class="container">
+      <div class="col-md-9" id="basket">
+        <div class="box">
+
+          <h1>Shopping cart</h1>
+          <p class="text-muted">You currently have <%= number_of_cart_items %> item(s) in your cart.</p>
+
+          <div class="table-responsive">
+            <table class="table">
+              <thead>
+                <tr>
+                  <th colspan="2">Product</th>
+                  <th>Quantity</th>
+                  <th>Unit price</th>
+                  <th>Discount</th>
+                  <th colspan="2">Total</th>
+                </tr>
+              </thead>
+
+              <tbody>
+              <% @cart_items.each do |cart_item| %>
+              <tr>
+                <td>
+                  <a href="#">
+                      <%= image_tag cart_item.product.photo_url %>
+                  </a>
+                </td>
+                <td><a href=<%="/products/#{cart_item.product.id}"%>><%=cart_item.product.name.capitalize%></a>
```

Comment:
From @lacuchilla:
Remember to add a space to separate the ERB tags and your code. It makes it a bit more readable Seeing this throughout the codebase)